packages/debugger/src/panels/header.ts

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { Toolbar } from '@jupyterlab/apputils';
+
+import {
+  ITranslator,
+  nullTranslator,
+  TranslationBundle
+} from '@jupyterlab/translation';
+
+import { PanelLayout, Widget } from '@lumino/widgets';
+
+import { caretDownEmptyIcon } from '@jupyterlab/ui-components';
+
+/**
+ * The base header for a debugger panels.
+ */
+export class PanelHeader extends Widget {
+  /**
+   * Instantiate a new PanelHeader.
+   */
+  constructor(translator?: ITranslator) {
+    super({ node: document.createElement('div') });
+    this.node.classList.add('jp-stack-panel-header');
+
+    translator = translator || nullTranslator;
+    this._trans = translator.load('jupyterlab');
+    this.titleWidget = new Widget({ node: document.createElement('h2') });
+
+    this._expandIcon = new Widget({ node: document.createElement('div') });
+
+    this._iconAngle = 0;
+    this._iconElement = caretDownEmptyIcon.element({
+      container: this._expandIcon.node
+    });
+    this._iconElement.classList.add(PanelHeader.ICON_EXPANDING_CLASS);
+
+    this._paddingDiv = new Widget({ node: document.createElement('div') });
+    this._paddingDiv.node.style.flexGrow = '1';
+    this._paddingDiv.node.style.height = '100%';
```

Comment:
Can we move this to a CSS file? I think the convention is to create a new class name matching the name of the class, so `jp-PanelHeader` and use appropriate selector in CSS styles file.